cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerConfigService.java

```diff
@@ -101,4 +105,38 @@ private void modifyKnoxAutorestart(ApiClient client, String clusterName, boolean
             return Optional.empty();
         }
     }
+
+    public void modifyHueKnoxProxyHosts(ApiClient client, String clusterName, String knoxProxyhosts) {
```

Comment:
seems this is really hue specific? can we do a little bit more generic solution here? I guess in the future there will be more